core/rpc/api/src/author/mod.rs

```diff
@@ -83,4 +83,16 @@ pub trait AuthorApi<Hash, BlockHash> {
 		metadata: Option<Self::Metadata>,
 		id: SubscriptionId
 	) -> Result<bool>;
+
+	/// Watch multiple extrinsics (own or from network)
+	#[pubsub(
+		subscription = "author_extrinsicUpdate",
+		subscribe,
+		name = "author_watchExtrinsic"
+	)]
+	fn track_extrinsic(&self,
```

Comment:
Maybe let's rename the function `watch_extrinsic` now and the other to `submit_and_watch` to avoid confusion.

Reply:
Yeah I was just reluctant about this long name